commands/command_prune.go

```diff
@@ -344,10 +347,10 @@ func pruneTaskGetPreviousVersionsOfRef(gitscanner *lfs.GitScanner, ref string, s
 		if err != nil {
 			errorChan <- err
 			return
-		} else {
-			retainChan <- p.Oid
-			tracerx.Printf("RETAIN: %v via ref %v >= %v", p.Oid, ref, since)
 		}
+
+		retainChan <- p.Oid
+		tracerx.Printf("RETAIN: %v via ref %v >= %v", p.Oid, ref, since)
```

Comment:
I guess aee2e19 is just my OCD 😄 
Let me know if you don't like this kind of stuff and I remove the commit from the PR!

Reply:
No, it's great! Thank you.